.gitignore

```diff
@@ -19,6 +19,7 @@ openlibrary.egg-info/
 pip-log.txt
 supervisord.pid
 vendor/apache-solr-1.4.0/
+vendor/infogami/
```

Comment:
It looks like we can achieve this through different means:
http://www.nils-haldenwang.de/frameworks-and-tools/git/how-to-ignore-changes-in-git-submodules#:~:text=One%20method%20is%20to%20provide,%22%2C%20which%20is%20the%20default.